js/data/bucket/symbol_bucket.js

```diff
@@ -29,6 +29,8 @@ function SymbolBucket(options) {
     this.showCollisionBoxes = options.showCollisionBoxes;
     this.overscaling = options.overscaling;
     this.collisionBoxArray = options.collisionBoxArray;
+    this.symbolQuadsBuffer = options.symbolQuadsBuffer;
+    this.symbolInstancesBuffer = options.symbolInstancesBuffer;
```

Comment:
Can we call these `Array`s instead of `Buffer`s? We generally use the term `Buffer` exclusively in the context of a WebGL buffer (though I'd like to phase that out too)